README.md

```diff
@@ -1,5 +1,13 @@
 ## RoMi-dashboard
 
+<h1>Badges</h1>
```

Comment:
no need for a badges header, just put them at the top before `## RoMi-dashboard`